editoast/src/views/timetable/stdcm.rs

```diff
@@ -251,7 +258,34 @@ async fn stdcm(
 
     let stdcm_response = stdcm_request.fetch(core_client.as_ref()).await?;
 
-    // 6. Handle STDCM Core Response
+    // 6. Check if the current tracing level is debug or lower, and if so, log STDCM request and response
+    if tracing::Level::DEBUG <= tracing::level_filters::LevelFilter::current() {
```

Comment:
Is there any advantage of using Yoda conditions in rust since boolean assignment is syntactically invalid?

Reply:
Oh and it seems the condition is reversed if I'm not mistaken

Reply:
Done

Reply:
Follow up: my bad, the condition isn't reversed. Indeed `TRACE > DEBUG > INFO > ...`, which is a bit unintuitive imo, especially that the log constant values are in the reverse order 😅 More info: https://docs.rs/tracing-core/0.1.33/src/tracing_core/metadata.rs.html#871